src/Symfony/Component/Notifier/Test/TransportTestCase.php

```diff
@@ -96,7 +96,13 @@ public function testCanSetCustomHost()
 
         $transport->setHost($customHost = self::CUSTOM_HOST);
 
-        $this->assertStringContainsString(sprintf('://%s', $customHost), (string) $transport);
+        $transportAsString = (string) $transport;
+
+        if (false == !strpos($transportAsString, '@')) {
```

Comment:
isn't it `false !== strpos`?

Reply:
https://github.com/symfony/symfony/pull/41985#discussion_r663685832 yes :-)